contents/docs/self-host/deploy/hobby.mdx

```diff
@@ -6,12 +6,26 @@ showTitle: true
 
 We provide a PostHog "hobby deployment" option for those that want to try out self-hosted PostHog without having to spend a lot on infrastructure costs.
 
-This type of deployment is aimed at users who want to test out the platform or do some minimal tracking (e.g. for a personal blog). It should **not** be used as a production instance for tracking a product with any reasonable amount of scale.
+This type of deployment is aimed at users who want to test out the platform or do some minimal tracking (e.g. for a personal blog, or to test PostHog as simply as possible with low volumes).
+
+It should **not** be used as a production instance for tracking a product with any reasonable amount of scale.
+
+## Pre-requisites
+
+* You have deployed a Linux Ubuntu Virtual Machine. 
```

Comment:
@fuziontech does this only work on Ubuntu? I thought I'd tested it on Debian - maybe I'm wrong.

Reply:
I guess they're quite similar though.

Reply:
I've only tested it on Ubuntu - and for now that's the only thing that's really supported. Adding Debian would be really easy - and not terribly difficult if we had users ask for it.

Reply:
Cool makes sense. Lets keep it as it is until we get feedback to suggest we do otherwise.